src/dosbox.cpp

```diff
@@ -597,19 +597,23 @@ void DOSBOX_Init(void) {
 	Pbool = secprop->Add_bool("nosound",Property::Changeable::OnlyAtStart,false);
 	Pbool->Set_help("Enable silent mode, sound is still emulated though.");
 
-	Pint = secprop->Add_int("rate",Property::Changeable::OnlyAtStart,44100);
+	Pint = secprop->Add_int("rate", Property::Changeable::OnlyAtStart, 44100);
 	Pint->Set_values(rates);
 	Pint->Set_help("Mixer sample rate, setting any device's rate higher than this will probably lower their sound quality.");
 
-	const char *blocksizes[] = {
-		 "1024", "2048", "4096", "8192", "512", "256", 0};
-	Pint = secprop->Add_int("blocksize",Property::Changeable::OnlyAtStart,1024);
+	const char *blocksizes[] = {"1024", "2048", "4096", "8192",
+	                            "512",  "256",  "128",  0};
+	Pint = secprop->Add_int("blocksize", deprecated, 1024);
```

Comment:
👍 nice use of deprecated!

What does the 1024 imply? Is only 1024 deprecated? Or if it just a dummy argument to deprecate all values?

Reply:
Just following convention I saw in the code for other deprecated values that still retained what appeared to be the default value.